dev/ci/README-users.md

```diff
@@ -5,7 +5,8 @@ You are encouraged to consider submitting your project for addition to
 Coq's CI. This means that:
 
 - Any time that a proposed change is breaking your project, Coq
-  developers and contributors will send you patches to adapt it or
+  developers and contributors will send you patches to adapt it (only
+  for plugins) or
```

Comment:
It can happen that patches are sent for libraries as well, so maybe change to:

```suggestion
  developers and contributors will send you patches to adapt it (systematically
  only for plugins) or
```